airflow/www/templates/airflow/dags.html

```diff
@@ -51,6 +51,14 @@
   {% for m in dashboard_alerts %}
     {{ message(m.message, m.category) }}
   {% endfor %}
+  {% for original_table_name, moved_table_name in migration_moved_data_alerts %}
+    {% call message(category='error', dismissable=false) %}
+      Airflow found incompatible data in the <code>{{ original_table_name }}</code> table in the
+      metadatabase, and have moved them to <code>{{ moved_table_name }}</code> during migration to
+      perform version upgrade. Please inspect the moved data to decide whether you need to keep them,
```

Comment:
```suggestion
      metadatabase, and has moved them to <code>{{ moved_table_name }}</code> during the database
      version upgrade. Please inspect the moved data to decide whether you need to keep them,
```

Nit, sounds better to my ear?

Reply:
I wasn't so sure about my own wording, this is nice.